_posts/2016-07-04-the-rush-age.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+name: The-Rush-Age
+title: The Rush Age
+date: 2016-07-04 10:00:00 +00:00
+author: Sergio Rodrigo Royo
+canonical:
+    name: my personal blog
+    href: http://srodrigo.me/?p=183
+image:
+    src: /assets/img/custom/blog/2016-07-04-the-rush-age/always-in-a-rush.jpg
+    attribution:
+      text: by Mayank Pandey
+      href: http://www.photo-street.com/2011/11/always-in-rush.html
```

Comment:
Do we have the rights for this image? I can't find any licensing info on the link